src/operations/get_more.ts

```diff
@@ -12,20 +12,35 @@ import { AbstractOperation, Aspect, defineAspects, OperationOptions } from './op
 export interface GetMoreOptions extends OperationOptions {
   /** Set the batchSize for the getMoreCommand when iterating over the query results. */
   batchSize?: number;
-  /** You can put a $comment field on a query to make looking in the profiler logs simpler. */
-  comment?: string | Document;
+  /**
+   * Comment to apply to the operation.
+   *
+   * getMore only supports 'comment' in server versions 4.4 and above.
+   */
+  comment?: any;
   /** Number of milliseconds to wait before aborting the query. */
   maxTimeMS?: number;
 }
 
+function filterOptions(
```

Comment:
We have a filterOptions in utils.ts, maybe we can reuse / update that one?

Reply:
Done!  Let me know what you think of my approach.  

The existing utility filtered based on an allowlist, whereas I needed a denylist.  I did think that it might be worthwhile to have since we'll probably filter objects like this in the future so I just added a second utility.